### DISCUSSION TITLE
问一下为啥这些全变成已过时了

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [AlistGo/alist#1567] [Q&A]
![屏幕截图+2022-08-26+203736](https://user-images.githubusercontent.com/73635883/186905394-c4d8a330-cc94-4f9d-aab7-e7011ac334f4.png)
在升级版本的时候,网站白屏,退回旧版本,备份,装上新版本,恢复,就这样了.没有在文档,或者别的地方搜索到类似问题

**Comment by xhofe** [comment]
重启即可。
